ext/RMagick/rmimage.c

```diff
@@ -2289,7 +2486,7 @@ Image_channel_mean(int argc, VALUE *argv, VALUE self)
 VALUE
 Image_channel_entropy(int argc, VALUE *argv, VALUE self)
 {
-#if defined(HAVE_GETIMAGECHANNELENTROPY)
+#if defined(HAVE_GETIMAGECHANNELENTROPY) || defined(IMAGEMAGICK_7)
```

Comment:
I wonder if it would be worth using `IMAGEMAGICK_GREATER_THAN_EQUAL_6_9_0` for these cases.

Reply:
Okay if we move this to a new PR? And maybe we could also replace `HAVE_SETIMAGEGRAY` with`IMAGEMAGICK_GREATER_THAN_EQUAL_6_9_2` there?

Reply:
Sure, sounds good.